frontend/src/components/NavBar.jsx

```diff
@@ -41,21 +51,25 @@ const NavBar = () => {
           </button>
           <div className="collapse navbar-collapse" id="navbarNav">
             <ul className="navbar-nav me-auto mb-2 mb-lg-0">
-              <li className="nav-item">
-                <Link className="nav-link" to={{ hash: "run-info" }}>
-                  Run Info
-                </Link>
-              </li>
-              <li className="nav-item">
-                <Link className="nav-link" to={{ hash: "summary" }}>
-                  Summary
-                </Link>
-              </li>
-              <li className="nav-item">
-                <Link className="nav-link" to={{ hash: "cases" }}>
-                  Details
-                </Link>
-              </li>
+              {isDialectPage ? (
```

Comment:
I think more common pattern is to use `isDialectPage && ...`, at least there is no null at the end